Frameworks/CoreText/CTFont.mm

```diff
@@ -249,25 +249,16 @@ CTFontRef CTFontCreateCopyWithAttributes(CTFontRef font, CGFloat size, const CGA
 }
 
 /**
- @Status Caveat
+ @Status Stub
  @Notes matrix parameter not supported
 */
 CTFontRef CTFontCreateCopyWithSymbolicTraits(
     CTFontRef font, CGFloat size, const CGAffineTransform* matrix, CTFontSymbolicTraits symTraitValue, CTFontSymbolicTraits symTraitMask) {
     if (font == nil) {
         return nullptr;
     }
-
-    UIFontDescriptorSymbolicTraits existingTraits = [(UIFont*)font fontDescriptor].symbolicTraits;
-    UIFontDescriptor* newFontDescriptor =
-        [[(UIFont*)font fontDescriptor] fontDescriptorWithSymbolicTraits:(existingTraits | symTraitValue) & symTraitMask];
-
-    if (size == 0.0f) {
-        size = ((UIFont*)font).pointSize;
-    }
-    UIFont* ret = [[_LazyUIFont fontWithDescriptor:newFontDescriptor size:size] retain];
-
-    return (CTFontRef)ret;
+    UNIMPLEMENTED();
```

Comment:
>UNIMPLEMENTED [](start = 4, length = 13)

Given we have CT/UIFont bridged we can implement parts of this API now? Are you tracking this with a different task?